travis/get_openapi_description.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# python3
+
+import os
+
+from absl import app
+
+from grr_response_server.gui import api_call_router
+from grr_response_server.gui.api_plugins import metadata as metadata_plugin
+
+OPENAPI_JSON_FOLDER_NAME = os.environ.get(
+  "OPENAPI_JSON_FOLDER_NAME",
+  default="generated_description"
+)
+OPENAPI_JSON_FILE_NAME = os.environ.get(
+  "OPENAPI_JSON_FILE_NAME",
+  default="openapi_description.json"
+)
```

Comment:
Using flags is a cleaner solution, I think. Also, not just make this whole thing take the direct path to the file to write to? Seems much simpler.

Reply:
Thank you for your suggestion! I replaced my usage of environmental variables with flags and I used the entire file path.